client/client.go

```diff
@@ -0,0 +1,105 @@
+package client
+
+import (
+	"errors"
+	"log"
+
+	pb "github.com/c-fs/cfs/proto"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+type Client struct {
+	conn *grpc.ClientConn
```

Comment:
why not keep the grpc client? we create the client for each function call right now.